pages/foundation/index.html

```diff
@@ -39,6 +39,11 @@ <h3>Community Member Sponsorship</h3>
           <p><a href="sponsorship.html#community">Learn more about Community Member Sponsorship</a></p>
             <p>See all the current supporters on our <a href="/foundation/sponsor-list.html">Sponsors List</a>.</p>
 
+            <h3>Tools and Infrastructure Partners</h3>
+            <p>The Grails Foundation is truly appreciative of the tremendous support it receives
+              from <a href="https://grails.org/foundation/sponsor-list.html">these organizations</a>. Their generous contributions of development tools, infrastructure,
```

Comment:
@sdelamo It looks like this will simply link to the page. Can you please add an anchor link, so it links directly to the "Tools and Infrastructure Partners" section the way the Micronaut link does?

Reply:
We don't have an id in that page. If it is something that really bothers you, I can add an id to the h2 of that page and create an anchor link here. Let me know if you want me to do that.

Reply:
Yes. Please do that. Thanks!